### DISCUSSION TITLE
Will there be support for Linux distros, like Debian?

### DISCUSSION THREAD

**Discussion by MesaBlack** [Tyrrrz/YoutubeDownloader#356] [General]
I have a Chromebook, which runs a version of Linux, and it has this developer mode that gives you a virtual console named Penguin, built upon the Debian environment. The current release of this application is definitely for Windows, as it consists of dlls. Those don't work on chrome os.

**Comment by Tyrrrz** [comment]
Most likely not because it would require a pretty significant rewrite. I'd recommend using https://github.com/yt-dlp/yt-dlp or something instead.
